packages/web-components/fast-foundation/src/menu-item/menu-item.ts

```diff
@@ -221,18 +222,33 @@ export class FASTMenuItem extends FASTElement {
                 this.expandAndFocus();
                 return false;
 
+            case keyEscape:
+                // close submenu
+                if (this.expanded) {
+                    this.closeSubMenu();
+                    return false;
+                }
+
             case keyArrowLeft:
                 //close submenu
                 if (this.expanded) {
-                    this.expanded = false;
-                    this.focus();
+                    this.closeSubMenu();
                     return false;
                 }
         }
 
         return true;
     };
 
+    /**
+     * @internal
+     */
+    public closeSubMenu = (): void => {
```

Comment:
Should this potentially be protected or private considering it's a key part of having accessible keyboard navigation? Protected would allow classes extending to do work here without allowing the complete overriding of the function itself.

Reply:
Yes, typo. My bad.